indra/llappearance/llpolymorph.h

```diff
@@ -183,7 +183,7 @@ class alignas(16) LLPolyMorphTarget : public LLViewerVisualParam
 	LLPolyVertexMask *				mVertMask;
 	ESex							mLastSex;
 	// number of morph masks that haven't been generated, must be 0 before this morph is applied
-	BOOL							mNumMorphMasksPending;
+	S32								mNumMorphMasksPending;
```

Comment:
Great! Thank you for addressing this one. I thought it had to be a S32 since it was clearly used as one, but skipped it in favour of bulk conversion.